src/LoginForm/LoginForm.js

```diff
@@ -87,14 +92,17 @@ export class LoginForm extends Component {
           onChange={this.updateChange}
           />
         </label>
-        
-        <button 
-        className="log-in-button" 
-        onClick={this.verifyCredentials} 
-        onSubmit={this.verifyCredentials} 
-        >Submit</button>
+
+        <Link to='/'
+          className="log-in-button" 
+          onClick={(event) => {
+            this.verifyCredentials(event)
+            }}
+        >Submit
+        </Link>
```

Comment:
the problem we were running into was that the button was wrapped around the link ! we just need to style this button